pkg/session/bootstrap.go

```diff
@@ -722,6 +722,25 @@ const (
 		GROUP BY table_schema, table_name, index_name
 		HAVING
 			sum(last_access_time) is null;`
+
+	// CreateIndexAdvisorTable is a table to store the index advisor results.
+	CreateIndexAdvisorTable = `CREATE TABLE IF NOT EXISTS mysql.index_advisor_results (
+       id bigint primary key not null auto_increment,
+       created_at timestamp not null,
+       updated_at timestamp not null,
+
+       schema_name varchar(64) not null,
+       table_name varchar(64) not null,
+       index_name varchar(127) not null,
+       index_columns varchar(500) not null COMMENT 'split by ",", e.g. "c1", "c1,c2", "c1,c2,c3,c4,c5"',
+
+       index_details json,        -- est_index_size, reason, DDL to create this index, ...
+       top_impacted_queries json, -- improvement, plan before and after this index, ...
+       workload_impact json,      -- improvement and more details, ...
+       extra json,                -- for the cloud env to save more info like RU, cost_saving, ...
+       index idx_create(created_at),
+       index idx_update(updated_at),
+       unique index idx(schema_name, table_name, index_columns))`
```

Comment:
if a table dropped and add same index using advisor again, will it conflict?

Reply:
Maybe, but we'll use `insert ... on duplicate ...` to update this table to make sure there is only one record for an unique index.